lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -169,6 +179,25 @@ class ReportUIFeatures {
     this._templateContext = context;
   }
 
+  /**
+   * @param {{text: string, icon?: string, onClick: () => void}} opts
+   */
+  createButton(opts) {
```

Comment:
`addButton` since it adds it to the dom? (from the name i didnt expect it to do the append too, but it makes sense)